easybuild/framework/easyconfig/tweak.py

```diff
@@ -166,7 +166,7 @@ def __repr__(self):
     for (key, val) in tweaks.items():
 
         if isinstance(val, list):
-            regexp = re.compile(r"^\s*%s\s*=\s*(.*)$" % key, re.M)
+            regexp = re.compile(r"^(?P<key>\s*%s)\s*=\s*(?P<val>.*)$" % key, re.M)
```

Comment:
no, this is exactly what fixes the issue where the newline above `toolchain` was disappearing...

tweaking is not about making the easyconfigs adher to the style policy, so if there was whitespace before a key, leave it there